test/test_linalg.py

```diff
@@ -1512,15 +1512,13 @@ def test_eig_reuse(self, device, dtype):
             atol = 1e-8
             rtol = 0
         self.assertEqual(X, Xhat, atol=atol, rtol=rtol, msg='VeV\' wrong')
-        self.assertFalse(v.is_contiguous(), 'V is contiguous')
```

Comment:
I removed this check because it was failing on CUDA before this PR (hence why this test was marked `@onlyCPU`), it is now failing on CPU, and it doesn't seem like this check is necessary anyway

Reply:
It should not be failing neither on cpu nor on cuda, you are allocating `v` as a contiguous tensor of correct size, and send it as `out`. The function is not supposed to resize or restride correctly-sized `out` arguments. I'm surprised that it happens, though, the code looks correct.

Reply:
Ah ok, I think I understand. This line was asserting that `v.is_contiguous()` returns False, meaning that the result should not be contiguous. If I change this to assertTrue, meaning that the result should be contiguous, it passes. So to me, it seems like we should keep this check, but change it to assertTrue

Reply:
Right, I forgot that the first port was of cuda eig, not cpu eig. Now it makes sense, please reenable the check with `assertTrue`.